testing/tests/test_e2e/test_pebble.py

```diff
@@ -481,8 +481,18 @@ def _on_check_failed(self, event):
             infos.append(event.info)
 
     ctx = Context(MyCharm, meta={"name": "foo", "containers": {"foo": {}}})
-    check = CheckInfo("http-check", failures=7, status=pebble.CheckStatus.DOWN)
-    container = Container("foo", check_infos={check})
+    layer = pebble.Layer({
```

Comment:
All the tests in this file are unchanged, except that they add in the layer that really ought to have already been there.